MockKit/MockCGMManager.swift

```diff
@@ -32,12 +32,15 @@ public final class MockCGMManager: TestingCGMManager {
         public let backgroundContent: DeviceAlert.Content
     }
     let alerts: [DeviceAlert.AlertIdentifier: MockAlert] = [
-        submarine.identifier: submarine, buzz.identifier: buzz
+        submarine.identifier: submarine, buzz.identifier: buzz, critical.identifier: critical
```

Comment:
[nit] Any particular ordering here. I see in `alerts` it is `submarine`, `buzz`, and `critical`, but down below they are defined as `submarine`, `critical`, `buzz`. If order is irrelevant, consider alphabetical (in case this gets larger it will be easier to find and be consistent).